### ISSUE TITLE
AWX_Task container do not processing correctly Git cookieFile option

### ISSUE THREAD

**Issue by kevit** [ansible/awx#5190]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
I have an enviroment with Git 2FA based on 3rd party issued cookie. Usually I am updating my .gitconfig file to place a right path to cookie there, but in awx_task 8.0.0 it does not work

##### ENVIRONMENT
* AWX version: 8.0.0
* AWX install method: docker on linux

##### STEPS TO REPRODUCE

1. Write a section in .gitconfig as

```
[http "http://example.org"]
cookieFile = /tmp/cookie.txt
```

2. start GIT_TRACE_CURL=1 git clone

##### EXPECTED RESULTS
I am expecting to see cookies inside curl request

##### ACTUAL RESULTS
Output contain no cookies in curl request

##### ADDITIONAL INFORMATION

curl version inside container 7.29.0, which reportedly having cookies processing issues

**Comment by kevit** [comment]
I did not check with the new Centos 8 container

**Comment by kevit** [comment]
I tried the official upgrade without much success

```
root@awx awxcompose]# docker-compose pull
Pulling memcached ... done
Pulling postgres  ... done
Pulling rabbitmq  ... done
Pulling web       ... done
Pulling task      ... done
[root@awx awxcompose]# docker-compose up -d --force-recreate --no-deps --build task
Recreating awx_task ... done
[root@awx awxcompose]# docker exec -it awx_task /bin/bash
```

There is still old version, probably because last image was updated 10 days ago and still contain centos 7

`https://hub.docker.com/layers/ansible/awx_task/latest/images/sha256-745ac9b36095650fc41497121803c5dfba93e6dec3f2420fe1eb39997d4cfb38
`

**Comment by ryanpetrello** [comment]
@kevit 

Does the problem go away if you disable bubblewrap?

https://docs.ansible.com/ansible-tower/latest/html/administration/proot_func_variables.html#bubblewrap-functionality-and-variables

If so, you'll need to add `/tmp/` to `AWX_PROOT_SHOW_PATHS` at `https://your-awx.example.org/api/v2/settings/jobs/`.

**Comment by kevit** [comment]
@ryanpetrello as I understood the problem is not related to AWX directly, but to particular setup of docker container (old curl version)

**Comment by ryanpetrello** [comment]
Okay, gotcha.  It's possible that the issue might go away with the latest version of AWX that has moved to CentOS 8 - https://github.com/ansible/awx/releases/tag/9.0.0

We're currently in the process of cutting this new release - mind giving it a try in an hour or two?

**Comment by ryanpetrello** [comment]
@kevit mind giving this a shot in 9.0.0?

**Comment by kevit** [comment]
@ryanpetrello yup, I'll do Monday morning

**Comment by kevit** [comment]
git-tools package missed in 9.0.0 image, some credentials helpers not there, f.e netrc
Moreover, 9.0.0 does not improve my situation, because another curl bug version 7.61* in https://github.com/curl/curl/issues/3299 , which was fixed in 7.63 https://curl.haxx.se/mail/lib-2018-12/0036.html
I am looking now for possible workaround

**Comment by kevit** [comment]
Thank you for participating, fortunately problem was related to cookie file format, for internal reason our internal software started to produce cookie file with spaces rather than tabs. I got few more gray hairs, but it resolved yet.
